Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;
+    padding: 10;
+}
+.cont1{
+    background-color: #90A4AE;
+    padding: 10px;

```

Comment:
The color value should be specified with a '#' before the hexadecimal value. For example, color: #515050;